deployments/forgeops/README.md

```diff
@@ -36,6 +36,77 @@ DS Password: zMO2W9IlOronDqrF2MtEha3Jiic3urZM
 =====================================================
 ```
 
+### Example
+The `things` endpoint in AM provides IoT-specific functionality. For example, a thing can request an OAuth 2.0 access token without having to know the credentials of the OAuth 2.0 client acting on its behalf.
+
+Before a thing can use the endpoint, it must be registered with the platform and have a valid session token. Firstly, use the `amadmin` credentials to register a thing identity in the platform. Save the `Password` returned in the connection details to a variable, for example:
```

Comment:
As discussed, we should register the thing through the Platform UI in order for it to only show up under **Things** and not under **Users**.